reark/src/main/java/io/reark/reark/pojo/IdentifiablePojo.java

```diff
@@ -0,0 +1,123 @@
+package io.reark.reark.pojo;
```

Comment:
I should also add that the value merging is only done in the networking process after fetching data, so the performance of the user facing application shouldn't be affected. Finally, the default behavior of mergeValues() is just to pick the new value (in ContentProviderStore), so turning this on is hopefully a conscious decision.